dashboard/views_test.py

```diff
@@ -175,6 +177,22 @@ def test_with_runs(self, mocked_refresh):
                     if course_data['runs'][0]['course_id'] == "course-v1:MITx+8.MechCX+2014_T1":
                         assert course_data['runs'][0]['status'] == CourseStatus.UPGRADE
 
+    @patch('backends.utils.refresh_user_token', autospec=True)
+    def test_certificates_enrollments(self, mocked_refresh):
+        """
+        Make sure we call the proper api functions
```

Comment:
More accurate: "Make sure we retrieve student certificates and enrollments whenever user dashboard data is fetched"